var/spack/repos/builtin/packages/sollve/package.py

```diff
@@ -0,0 +1,730 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sollve(CMakePackage):
+    """The SOLLVE Project is a collection of modular and reusable compiler and
+       toolchain technologies. Despite its name, SOLLVE has little to do
+       with traditional virtual machines, though it does provide helpful
+       libraries that can be used to build them. The name "LLVM" itself
+       is not an acronym; it is the full name of the project.
+    """
+
+    homepage = 'https://github.com/SOLLVE/' 
```

Comment:
```suggestion
    homepage = 'https://github.com/SOLLVE/sollve' 
```

Reply:
This package uses repositories from in https://github.com/SOLLVE, (e.g., https://github.com/SOLLVE/llvm, /clang, ...), so https://github.com/SOLLVE should be more correct in my understanding. However, if https://github.com/SOLLVE/sollve is still better, I will follow your suggestion.